telemetry/attributes.ts

```diff
@@ -0,0 +1,103 @@
+import { URL } from 'url';
+
+export interface TelemetryAttribute {
+  name: string;
+}
+
+export class TelemetryAttributes {
+  static fgaClientRequestClientId: TelemetryAttribute = { name: 'fga-client.request.client_id' };
```

Comment:
Do we need the indirection through name here? Can't it be a string?

Reply:
I was thinking that we might want to expand on our per-attribute reconfigurability later on, and this would allow us to do so without introducing breaking changes down the road; for that reason, I implemented this in the same manner across Python and Go.